spec/models/page_view_spec.rb

```diff
@@ -6,6 +6,13 @@
   it { is_expected.to belong_to(:user).optional }
   it { is_expected.to belong_to(:article) }
 
+  # rubocop:disable RSpec/AnyInstance
+  def mock_random_method
```

Comment:
Here, I am not sure how to mock rand method, so I am using `AnyInstance`, which, by telling from `rubocup`, is bad. Any different idea is welcome...